Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM fedora
+
+WORKDIR /build
+COPY . .
+
+ENV PY_COLORS=1
+
+RUN dnf -y upgrade openssl-libs
+RUN dnf -y install yum $(cat test-deps-fedora.txt)
```

Comment:
Wouldn't it be better to just remove `test-deps-fedora.txt` file and merge its contents into Dockerfile?